pylot/prediction/messages.py

```diff
@@ -1,6 +1,21 @@
+from typing import NamedTuple
 import erdos
 
 
+class PredictionMessageTuple(NamedTuple):
+    """
+    Message class to be used to send obstacle predictions.
+
+    Attributes:
+        timestamp (:py:class:`erdos.timestamp.Timestamp`): The timestamp of
+            the message.
+        predictions (list(:py:class:`~pylot.prediction.obstacle_prediction.ObstaclePrediction`)):  # noqa: E501
+            Obstacle predictions.
+    """
+    timestamp: erdos.Timestamp
+    predictions: list
+
+
```

Comment:
We can delete this and the `PredictionMessage`. The output streams of the prediction operators should only send lists of `ObstaclePrediction`, and don't need to be timestamped since the timestamp is already associated with the message.